src/ports/rt-kernel/pnal_eth.c

```diff
@@ -25,6 +25,10 @@
 static struct netif * interface[MAX_NUMBER_OF_IF];
 static int nic_index = 0;
 
+struct netif * os_eth_get_interface (void)
```

Comment:
I think we should use `pnal_eth_get_interface()` instead. (I know that other functions not yet are renamed)

Reply:
Removed function for now.